e2e/pagination/pagination.e2e-spec.ts

```diff
@@ -13,14 +14,17 @@ interface returnState {
 
 async function paginationState(paginationPO: PaginationPO) {
 	return paginationPO.locatorRoot.evaluate((rootNode: HTMLElement) => {
-		const returnState: returnState = {rootClasses: [], disabled: null, pages: []};
+		const returnState: returnState = {rootClasses: [], disabled: null, pages: [], hrefs: []};
 		const pagesElements = [...rootNode.querySelectorAll('.au-page')] as HTMLLinkElement[];
 		const pages = [];
+		const hrefs = [];
 		for (const element of pagesElements) {
+			hrefs.push(element.getAttribute('href') || ''.trim());
```

Comment:
Missing parenthesis, otherwise not sure that the `trim` is applied to your attribute. But is it necessary to trim here ?

Reply:
I think you are right in both cases. :)
I will remove it.